test/unit_test/deepdoc/parser/test_get_text_encoding.py

```
@@ -0,0 +1,118 @@
+    p = tmp_path / name
+    p.write_bytes(data)
+    return str(p)
+
+
+@pytest.mark.p2
+def test_reads_gbk_encoded_file_from_disk(get_text, tmp_path):
+    # A GBK-encoded text file (common for Chinese documents) is not valid UTF-8.
+    # Reading it via the file path used to open() in the platform-default text
+    # mode and raise UnicodeDecodeError; it must now be decoded through
+    # find_codec and round-trip to the original text.
+    text = "你好，世界"
```

Ruff (0.15.21)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)